h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -94,7 +97,8 @@ private GridSearch(Key<Grid> gkey, HyperSpaceWalker<MP, ?> hyperSpaceWalker, fin
     // this function the grid object is in DKV and accessible.
     final Grid<MP> grid;
     Keyed keyed = DKV.getGet(_result);
-    if (keyed != null) {
+
+    if (_isMultiRunSearch = keyed != null) {
```

Comment:
hidden assignment in if statemenet

Reply:
@honzasterba this is a valid usage of Java and I have seen examples in our codebase. I'm ok to change this but  I would like to ask team's opinion on this...whether we are ok with this syntax? Couple of times I run into situation when reviewers were suggesting opposite things about code conventions. I would prefer to discuss such cases and stick to our team's conventions. @michalkurka @seb-h2o @Pscheidl ( like if you are ok with the syntax and dislike if it is better to avoid it)